src/Filetype.js

```diff
@@ -33,9 +41,33 @@ export default class Filetype extends React.Component {
         }
     }
 
+    getAlert() {
+    alert('getAlert from Child');
+  }
+
+
+    check(event){
+        console.log("filtype "+this.props.filetype);
+        console.log("Les formats",this.props.formats);
```

Comment:
These `console.log` should be useful only during debugging.
And thus, not present in a Request For Review, which aims to be merged into master branch.

There are still `console.log`s in other files too.